core/src/it/scala/org/ensime/core/StructureViewBuilderSpec.scala

```diff
@@ -0,0 +1,110 @@
+package org.ensime.core
+
+import org.ensime.fixture._
+import org.ensime.api._
+import org.scalatest._
+import scala.collection.mutable.ListBuffer
+
+class StructureViewBuilderSpec extends WordSpec with Matchers
```

Comment:
can you please use `FlatSpec` and the `shouldBe` pattern? We're trying to move away from `WordSpec` and `assert`